<launch>
  <arg name="map" value="icra2020"/>
  <master auto="start"/>
  <param name="/use_sim_time" value="true"/>

  <!-- Run the map server -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(find hero_multistage)/maps/$(arg map).yaml" respawn="false"/>

  <!-- Run the stage simulator -->
  <node pkg="stage_ros" type="stageros" name="stageros" args="$(find hero_multistage)/worlds/$(arg map).world" respawn="false" >
    <remap from="robot_0/base_scan" to="robot_0/scan" />
    <remap from="robot_1/base_scan" to="robot_1/scan" />
    <remap from="robot_2/base_scan" to="robot_2/scan" />
    <remap from="robot_3/base_scan" to="robot_3/scan" />
    <param name="base_watchdog_timeout" value="0.01" />
  </node>

</launch>
